### ISSUE TITLE
Compatibility with LiteDB from python [QUESTION]

### ISSUE THREAD

**Issue by raulmd13** [litedb-org/LiteDB#2214]
Hi, I'm trying to read liteDB files generated by this with the litedb developed for python.
For what I' have seen, this two litedb have exactly the same name but seem to be two different things?

It's posible to read .db files from the .NET implementation in the python one?

**Comment by Garulf** [comment]
I have run into this issue as well. Unfortunately all my google searches confuse the two.

**Comment by AdrianSkib** [comment]
@raulmd13 did you ever figure this out?

**Comment by raulmd13** [comment]
Yeah, I've contacted both developers and these 2 projects are completely different besides having the same name.

**Comment by JKamsker** [comment]
As you found out, they're unrelated projects that happen to share a name. The Python `litedb` package can't read files written by this .NET library. The LiteDB file format (8 KB pages, a header starting with `** This is a LiteDB file **`, BSON documents, plus a `-log.db` write-ahead log) is only implemented by the .NET library.

To get the data into Python, the simplest route is to export it from .NET, or from LiteDB.Studio with the same SQL:

```csharp
using var db = new LiteDatabase(@"C:\data\my.db");
db.Execute("SELECT $ INTO $file('C:/data/mycollection.json') FROM mycollection");
// or CSV: $file('C:/data/mycollection.csv')
```

Then read the file with `json.load` in Python. If you need live access instead, you have to host the .NET library from Python through a .NET bridge.

Closing this as answered — if it doesn't cover your case, comment here and we'll reopen it.
